tests/core/contracts/test_contract_buildTransaction.py

```diff
@@ -1,19 +1,43 @@
-# -*- coding: utf-8 -*-
-
 import pytest
 
 from eth_utils.toolz import (
     dissoc,
 )
 
+from web3 import Web3
 from web3.exceptions import (
     ValidationError,
 )
+from web3.middleware import (
+    local_filter_middleware,
+)
+from web3.providers.eth_tester import (
+    EthereumTesterProvider,
+)
+
+# -*- coding: utf-8 -*-
+
 
 # Ignore warning in pyethereum 1.6 - will go away with the upgrade
 pytestmark = pytest.mark.filterwarnings("ignore:implicit cast from 'char *'")
 
 
+# Ignore warning in pyethereum 1.6 - will go away with the upgrade
+pytestmark = pytest.mark.filterwarnings("ignore:implicit cast from 'char *'")
+
+
+@pytest.fixture(
```

Comment:
I think this is a different web3 than what we want in the contract tests. Do tests fail if you remove this? Or do any warnings come back?

Reply:
No failure, and no new warnings! :)